cdap-gateway/src/test/java/co/cask/cdap/gateway/router/RouterResource.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
this was moved out from previous NettyRouterPipelineTest, plus added an option to pass in additional configuration.